tools/ci_testing/addons_cpu.sh

```diff
@@ -24,43 +24,18 @@ if [ "$1" != "--no-deps" ] && [ "$1" != "" ]; then
   exit 1
 fi
 
-# Make sure we're in the project root path.
-SCRIPT_DIR=$( cd ${0%/*} && pwd -P )
-ROOT_DIR=$( cd "$SCRIPT_DIR/../.." && pwd -P )
-cd $ROOT_DIR
-if [[ ! -d "tensorflow_addons" ]]; then
-    echo "ERROR: PWD: $PWD is not project root"
-    exit 1
-fi
-
-PLATFORM="$(uname -s | tr 'A-Z' 'a-z')"
-
-if [[ ${PLATFORM} == "darwin" ]]; then
-    N_JOBS=$(sysctl -n hw.ncpu)
-else
-    N_JOBS=$(grep -c ^processor /proc/cpuinfo)
-fi
-
-echo ""
-echo "Bazel will use ${N_JOBS} concurrent job(s)."
-echo ""
-
 export CC_OPT_FLAGS='-mavx'
 export TF_NEED_CUDA=0
 
 # Check if python3 is available. On Windows VM it is not.
 if [ -x "$(command -v python3)" ]; then
-    python3 ./configure.py $1
-  else
-    python ./configure.py $1
+  PYTHON_BINARY=python3
+else
+  PYTHON_BINARY=python
 fi
 
+$PYTHON_BINARY -m pip install -r tools/tests_dependencies/pytest.txt
+$PYTHON_BINARY ./configure.py $1
 cat ./.bazelrc
-
-## Run bazel test command. Double test timeouts to avoid flakes.
-${BAZEL_PATH:=bazel} test -c opt -k \
-    --jobs=${N_JOBS} --test_timeout 300,450,1200,3600 \
-    --test_output=errors --local_test_jobs=8 \
-    //tensorflow_addons/...
-
-exit $?
+bash tools/install_so_files.sh
+$PYTHON_BINARY -m pytest --cov=tensorflow_addons -v --durations=25 -n auto ./tensorflow_addons
```

Comment:
Could we test with custom_ops and python_op fallbacks? That can be a seperate PR but want to debug to make sure they both work.

Reply:
That should be done at the op level, not really at the project-wide tests level. Since we have python variables to control that, we should have no problems testing the fallback in individual tests. 

I can open a separate pull request as I believe this is independant from this PR